### DISCUSSION TITLE
Is anyone using Renovate with a JavaScript monorepo managed by something other than Lerna?

### DISCUSSION THREAD

**Discussion by richardkazuomiller** [renovatebot/renovate#13818] [Suggest an Idea]
My project is using Lerna and Renovate. Since Lerna is unmaintained and will not accept any fixed or dependency updates, we are considering switching to singing like Rush or Turborepo, but I want to make sure whatever we choose works with Renovate. Are there any examples?

**Comment by viceice** [comment]
No, but you can create a small sample repo to test.

Report any issues, so we can see what to do.

**Reply by rarkins** [reply]
Vanilla Yarn Workspaces works fine too

**Reply by richardkazuomiller** [reply]
Thanks. I was hoping that I could stand on the shoulders of someone who thought of doing this before me but I will try to find the time to set this up 😅

**Comment by h-parker-higgins** [comment]
Looks like there's an opportunity for the rushstack team at Microsoft to provide a plugin here. It's currently a Low Priority feature request: https://github.com/microsoft/rushstack/issues/3388
